.github/workflows/test.yml

```diff
@@ -47,7 +47,7 @@ jobs:
             --ignore=tests/models/test_metatensor.py
 
       - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v5
+        uses: codecov/codecov-action@v5.4.0
```

Comment:
[v5.4.2](https://github.com/codecov/codecov-action/releases/tag/v5.4.2) is out so we can revert to v5 to get later releases

Reply:
Great, I can sneak this in to a future PR